src/parse/import.rs

```diff
@@ -7,6 +7,56 @@ use crate::{common::QuoteKind, error::SassResult, lexer::Lexer, value::Value, To
 
 use super::{Parser, Stmt};
 
+/// Searches the current directory of the file then searches in load paths directories
+/// if the import has not yet been found.
+/// [https://sass-lang.com/documentation/at-rules/import#finding-the-file](finding a file)
+/// [https://sass-lang.com/documentation/at-rules/import#load-paths](load path)
+fn find_import(file_path: &PathBuf, name: &OsStr, load_paths: &[&Path]) -> Option<PathBuf> {
+    let paths = [
+        file_path.with_file_name(name).with_extension("scss"),
+        file_path
+            .with_file_name(format!("_{}", name.to_str().unwrap()))
+            .with_extension("scss"),
+        file_path.clone(),
+        file_path.join("index.scss"),
+        file_path.join("_index.scss"),
+    ];
+
+    for name in &paths {
+        if name.is_file() {
+            return Some(name.to_path_buf());
+        }
+    }
+
+    for path in load_paths {
```

Comment:
Also, I think we could do `paths.chain(load_paths)` and do `find` only once. In my mind

```rust
let paths = [...];
let load_paths = load_paths.iter().flat_map(|p| if p.is_dir() { &[...] } else { &[...] }); // or use Itertools::partition
paths.iter().chain(load_paths).find(Path::is_file)
```

We could also use `Iterator::find` as a better API.